sup3r/pipeline/forward_pass.py

```diff
@@ -386,7 +386,9 @@ def get_node_cmd(cls, config):
     @classmethod
     def _constant_output_check(cls, out_data, allowed_const):
         """Check if forward pass output is constant. This can happen when the
-        chunk going through the forward pass is too big.
+        chunk going through the forward pass is too big. This is due to a
+        tensorflow padding bug, with the padding mode set to 'reflect'.
+        https://github.com/tensorflow/tensorflow/issues/91027
```

Comment:
You might note that in our current TF version 2.15 this actually just results in scrambled outputs.